docs/storage.md

```diff
@@ -114,6 +114,7 @@ a set of interfaces that allow integrating with remote storage systems.
 Prometheus integrates with remote storage systems in two ways:
 
 * Prometheus can write samples that it ingests to a remote URL in a standardized format.
+* Prometheus can receive samples from other Prometheus servers in a standardized format.
 * Prometheus can read (back) sample data from a remote URL in a standardized format.
 
 ![Remote read and write architecture](images/remote_integrations.png)
```

Comment:
The diagram would need an update?

Reply:
I would not change that while it is experimental

Reply:
looking at @juliusv maybe

Reply:
I asked this same question ~1y ago, they were made with draw.io IIRC.

Reply:
Hah! That's an ugly old-style drawing by me indeed. I'd suggest making a new and somewhat prettier one (with source XML stored next to the PNG/SVG so people can update it, like I did in https://github.com/prometheus/prometheus/tree/master/documentation/images) using https://app.diagrams.net/ (rename of draw.io). But I also think we should wait until it's not experimental anymore.